spinalcordtoolbox/download.py

```diff
@@ -32,7 +32,8 @@
     },
     "sct_testing_data": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/sct_testing_data/releases/download/r20210330230310/sct_testing_data-r20210330230310.zip",
+            # FIXME: This link should NOT be merged to master. It should be replaced via a new sct_testing_data release.
+            "https://github.com/spinalcordtoolbox/sct_testing_data/releases/download/r20210330230310/sct_testing_data-PR3977.zip",
```

Comment:
Reminder for myself to merge the `sct_testing_data` PR, create a new release, then update this link.

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/28ad78bfeefb883d705068475935f18734ea4b50.